src/trivyHelper.ts

```diff
@@ -58,12 +60,16 @@ export async function runTrivy(): Promise<TrivyResult> {
 }
 
 export async function getTrivy(): Promise<string> {
-    const latestTrivyVersion = await getLatestTrivyVersion();
 
-    let cachedToolPath = toolCache.find(trivyToolName, latestTrivyVersion);
+    let version = inputHelper.trivyVersion;
+    if(version == 'latest'){
+        version = await getLatestTrivyVersion();
+    }
+    console.log(`Use Trivy version: ${version}` );
```

Comment:
Sorry I missed this earlier. My bad.

```suggestion
    core.debug(`Use Trivy version: ${version}`);
```